pkgs/applications/networking/cluster/fluxcd/default.nix

```diff
@@ -1,26 +1,38 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+{ lib, buildGoModule, fetchFromGitHub, fetchzip, installShellFiles }:
+
+let
+  fluxVersion = "0.10.0";
+
+  manifests = fetchzip {
+    url = "https://github.com/fluxcd/flux2/releases/download/v${fluxVersion}/manifests.tar.gz";
+    sha256 = "Der1Ud27eIV450KkxDTF2frmeKEHKsg6vJgdXE+3548=";
+    stripRoot = false;
+  };
```

Comment:
I think this is not needed

Reply:
The manifests are now embedded in the Flux binary since v0.9.0.

Reply:
https://github.com/fluxcd/flux2/pull/987

Reply:
I see, I got confused because there is a manifests directory in the src, but it's not the same content,